vunit/sim_if/ghdl.py

```diff
@@ -364,3 +381,67 @@ def simulate(  # pylint: disable=too-many-locals
             subprocess.call(cmd)
 
         return status
+
+    def _compile_source_file(self, source_file, printer):
+        """
+        Runs parent command for compilation, and moves the .gcno file (if any) to the compilation output folder
```

Comment:
Very long line. Not sure what vunit coding guidelines say but it is too long for a side-by-side diff on my screen.

Reply:
Vunit is ok, but I'll change it anyway

Reply:
Done